package.json

```diff
@@ -224,4 +224,4 @@
       "npm run lint:fix"
     ]
   }
-}
+}
```

Comment:
please revert this file

Reply:
these are the files which have been modified.
  modified:   test/e2e/features/cat-ce/10-pass_with_no_faults.feature
        modified:   test/e2e/features/cat-mod1/01-pass-journy.feature
        modified:   test/e2e/features/cat-mod1/02-fail-journy.feature
        modified:   test/e2e/features/cat-mod1/03-rekey.feature
        modified:   test/e2e/features/cat-mod1/04-autosave.feature
        modified:   test/e2e/step-definitions/testreport-steps.ts
packge.json line 227 is showing as }

Reply:
commit and pushed again